scripts/ij.bazelproject

```diff
@@ -1,14 +1,14 @@
 # Setup IntelliJ for Bazel development using the IntelliJ Bazel Plugin.
 # See https://github.com/bazelbuild/intellij for installation instructions.
 directories:
-  src/main
-  src/test
+  .
```

Comment:
I tried, but it doesn't change anything. The directories still show up in the project directory list, but as they're already "excluded directories" (marked as orange folders) by default, adding it here is a no-op. Maybe the IJWB plugin automatically excludes symlinks or bazel-*?